modules/monitoring-configuring-metrics-collection-profiles.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * monitoring/configuring-the-monitoring-stack.adoc
+
+:_content-type: CONCEPT
+[id="configuring-metrics-collection-profiles_{context}"]
+= Configuring metrics collection profiles
+ 
+[IMPORTANT]
+====
+[subs="attributes+"]
+Metrics collection profiles are a Technology Preview feature only. Technology Preview features are not supported with Red Hat production service level agreements (SLAs) and might not be functionally complete.
+Red Hat does not recommend using them in production. 
+These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.
+
+For more information about the support scope of Red Hat Technology Preview features, see link:https://access.redhat.com/support/offerings/techpreview[https://access.redhat.com/support/offerings/techpreview].
+====
+
+By default, Prometheus collects a large amount of time-series metrics data from default metrics targets.
+Using a metrics collection profile for core {product-title} monitoring components, you can choose to collect either the default amount of metrics data or a minimal amount of data. 
+By choosing to collect a minimal amount of metrics data, you can lower the amount of CPU and memory resources used by the monitoring stack while simultaneously maintaining basic monitoring features such as alerting.
+
+[id="about-metrics-collection-profiles_{context}"]
+== About metrics collection profiles
+
+You can choose one of two metrics collection profiles:
+
+* *Full*: Prometheus collects all metrics data exposed by the different platform components.
+* *Minimal*: Prometheus only collects the metrics data required for platform alerts, recording rules, telemetry and console dashboards.
```

Comment:
Maybe we should add something saying that when using a profile other than "Full" some glitches might show up at the console level, but that no metrics used in alerts are ever discarded.
cc @simonpasquier

Reply:
If we include information like this, it should be at the most generic level. But I'd prefer not to include anything about console error messages unless absolutely necessary. That said, I'll defer to @simonpasquier's guidance here.

Reply:
@simonpasquier What's your take on including a note that console errors or glitchiness might occur, but no metrics used in alerts are ever discarded? My preference is to leave information like this out of the core docs and include it as a known issue in the release notes if necessary. WDYT?

Reply:
I'm going to make a call and say that any user-facing console error messages or bugs should be reported as known issues in the Release Notes. This is a TP feature, so users are already warned that the feature might have some glitches. So I am setting this to resolved.